db/migrate/20201208093803_change_activity_programme_status_to_enum.rb

```diff
@@ -0,0 +1,25 @@
+class ChangeActivityProgrammeStatusToEnum < ActiveRecord::Migration[6.0]
+  def up
+    add_column :activities, :programme_status_integer, :integer
+
+    Activity.all.each do |activity|
+      programme_status = activity.programme_status.to_i
+      activity.update_attribute(:programme_status_integer, programme_status)
+    end
+
+    remove_column :activities, :programme_status, :string
+    rename_column :activities, :programme_status_integer, :programme_status
+  end
+
+  def down
+    add_column :activities, :programme_status_string, :string
+
+    Activity.all.each do |activity|
+      programme_status = "%02d" % activity.programme_status
```

Comment:
As per above, should those current `0` programme statuses be changed back to `nil` here?